packages/shared/README.md

```diff
@@ -0,0 +1,79 @@
+# browserd/stream-provider
+
+Represents the component that streams content to the client 🤕☁✨
+
+[![Build Status](https://dev.azure.com/bengreenier/browserd/_apis/build/status/stream-provider?branchName=repo-restructure)](https://dev.azure.com/bengreenier/browserd/_build/latest?definitionId=9&branchName=repo-restructure)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=bengreenier_browserd&metric=alert_status)](https://sonarcloud.io/dashboard?id=bengreenier_browserd)
```

Comment:
should be removed because it does not point at the quality check for the module alone